packages/calling-component-bindings/src/videoGallerySelector.ts

```diff
@@ -61,7 +61,7 @@ export type VideoGallerySelector = (
 export const videoGallerySelector: VideoGallerySelector = createSelector(
   [
     getScreenShareRemoteParticipant,
-    getRemoteParticipantsExcludingConsumers,
+    getRemoteParticipants,
```

Comment:
Why was this change needed, I believe we intentionally are filtering out consumers

Reply:
I see. It got refactored with merge conflicts. Updating it. Thanks :)